### PR TITLE
Remove remote connection aliases from compsys

### PR DESCRIPTION
**Pull request by belljustin**
Using the robot name as the hostname allows us to switch the robot's IP
easily without finding all the scripts that rely on the IP

### CODE REVIEW

**Comment by anassinator** [comment]
The only issue I see with this  is that `${ROBOT}` isn't added to `/etc/hosts` by `compsys`.
So, we should either document it in the README or add it to the install before merging this.

**Comment by belljustin** [comment]
^^ discussed this with @Baycken. We added it to [auv setup](https://github.com/mcgill-robotics/auv/pull/359/files#diff-6aa367da3fdfc952b15def7759aa5f0bR162) and decided against adding it to compsys because if someone does `rosworkon ...` they won't have the appropriate hostname in their `/etc/hosts` file

Adding it the README is probably a good idea

**Comment by anassinator** [comment]
I don’t think you meant to push some of these changes...

**Comment by belljustin** [comment]
Oh man... nope. I'll rebase 😛

File: README.md [hunk 1]
```diff
@@ -66,6 +66,20 @@ repository exists when the script runs, the robot specific setup will not run,
...
+#### robot hostnames
+
+Some aliases - e.g. `robotx`, `robotsh` - rely on the existence of a hostname
+matching the robot. In other words, when networking with a robot, make sure it
+has an entry in your `/etc/hosts` file.
+
+```
+# /etc/hosts
+
+10.10.10.10 auv
```
Comment (anassinator) [inline README.md]: curious why you guys changed this

File: aliases [hunk 2]
```diff
@@ -3,8 +3,8 @@
 # Initialize McGill Robotics aliases and functions
 
 # Connecting to robot
-alias robosh='mosh robotics@10.0.0.1'
-alias robotx='ssh -Y robotics@10.0.0.1'
+alias robosh="mosh robotics@${ROBOT}"
+alias robotx="ssh -Y robotics@${ROBOT}"
```
Comment (anassinator) [inline aliases]: I'm actually thinking we should just remove all of this from compsys. You can add it back to your project's `robotrc`

Comment (belljustin) [reply aliases]: Yeah - that makes sense

**Comment by anassinator** [comment]
What's the status here?

**Pull request closed without merging**